core/src/main/java/org/elasticsearch/action/index/IndexResponse.java

```diff
@@ -82,7 +83,8 @@ public String toString() {
         builder.append(",type=").append(getType());
         builder.append(",id=").append(getId());
         builder.append(",version=").append(getVersion());
-        builder.append(",created=").append(created);
+        builder.append(",created=").append(isCreated());
```

Comment:
Same feedback as the last `toString` - I think you can remove `created` and might want to look at `Operation`'s `toString`.